### ISSUE TITLE
Incorrect case for UInt8 causes it to miss highlighting the type

### ISSUE THREAD

**Issue by Psychlist1972** [madskristensen/MIDL#1]
First, I'll say that this makes editing MIDL 3.0 much less of a chore. Thank you. I'm still fighting how Visual Studio wants to indent MIDL, but this took a lot of the sting out of it.

Now, for the bug:
Parser has Uint8 instead of UInt8 in the regex. So it fails to highlight the type.

(from a quick look, it also appears to be missing the Char16 type, but I haven't used that type)

Pete

**Comment by madskristensen** [comment]
Thanks Pete!

**Comment by madskristensen** [comment]
Grab the [latest CI build](https://www.vsixgallery.com/extension/MIDL.148cb49f-0d60-482a-a9ca-800f2b6c579b) to get the fix

**Comment by Psychlist1972** [comment]
Thanks. I assume it takes a few for the CI build to get the bits to the VSIX gallery. Right now, it says "The resource you are looking for has been removed, had its name changed, or is temporarily unavailable."

**Comment by Psychlist1972** [comment]
Hmm. Still shows that error. Looks like the CI failed, maybe.

**Comment by Psychlist1972** [comment]
Hi Mads. When you get back in, can you update the gallery? Seems like the CI build broke it. 

It points to this, which has moved or been removed.

https://www.vsixgallery.com/extensions/MIDL.148cb49f-0d60-482a-a9ca-800f2b6c579b/WinRT%20Tools%20for%20C++%20v1.0.25.vsix

**Comment by madskristensen** [comment]
I'm pushing a new version to the Marketplace now. It will take a few minutes for the Marketplace to verify before it goes live

**Comment by Psychlist1972** [comment]
I got it installed. Thanks.

Not sure why the gallery link fails. Clicking download here:
https://www.vsixgallery.com/extension/MIDL.148cb49f-0d60-482a-a9ca-800f2b6c579b

Gets me:
"The resource you are looking for has been removed, had its name changed, or is temporarily unavailable."

I thought maybe it was my pi.hole blocking it, so I disabled that and it still failed. The version number seems correct (1.0.26 now)

So I was able to download it from the marketplace link instead. I tried to install that, but the installer complained it was already installed. Turns out it must have auto-updated earlier today as I already have 1.0.26 now, installed today. :)

Thanks!
